### PR TITLE
Configurable cassy version

### PR DESCRIPTION
**Pull request by tei-k**
# Description
https://scalar-labs.atlassian.net/browse/DLT-5754

### CODE REVIEW

File: examples/aws/cassandra/example.tfvars [hunk 1]
```diff
@@ -20,6 +20,7 @@ cassandra = {
 }
 
 cassy = {
+  # version                   = "v1.0.0"
```
Comment (tei-k) [inline examples/aws/cassandra/example.tfvars:23]: ❓ `image_tag` is would be better?

Comment (ymorimo) [reply examples/aws/cassandra/example.tfvars:23]: I think `image_tag` would correspond to scalardl module.

Comment (tei-k) [reply examples/aws/cassandra/example.tfvars:23]: 🙆‍♂️I think so too. fixed in 22cdb06.

Comment (feeblefakie) [reply examples/aws/cassandra/example.tfvars:23]: Should this example be updated ?

Comment (tei-k) [reply examples/aws/cassandra/example.tfvars]: oh! Updated c62ab08

**Pull request merged**